source/github-helpers/github-url.ts

```diff
@@ -1,8 +1,9 @@
 import {getCurrentBranch} from '.';
 
-function disambiguateReference(ambiguousReference: string[]): {branch: string; filePath: string} {
+function disambiguateReference(ambiguousReference: string[], search: string): {branch: string; filePath: string} {
 	const branch = ambiguousReference[0];
-	const filePath = ambiguousReference.slice(1).join('/');
+	const filePathFromSearch = decodeURI(search).split(/&path\[]=/).splice(1).join('/');
```

Comment:
Use `this.searchParams` to parse the file path safely:
```suggestion
	const filePathFromSearch = this.searchParams.getAll('path[]').join('/');
```
If the above is not falsey, return instantly
```suggestion
	const filePathFromSearch = this.searchParams.getAll('path[]').join('/');
	if (filePathFromSearch) {
		return {branch, filePath: filePathFromSearch};
	}
```